### DISCUSSION TITLE
How are you able to sustain this project since it's free?

### DISCUSSION THREAD

**Discussion by luciannuta** [danny-avila/LibreChat#4361] [General]
Hi there. How are you able to sustain this project since it's free?

**Comment by fuegovic** [comment]
You can help by sponsoring: https://github.com/sponsors/danny-avila

**Reply by luciannuta** [reply]
Ok, that's a good thing

**Comment by rio100** [comment]
> Hi there. How are you able to sustain this project since it's free?

He gets sponsors, however mostly passion I would imagine. Something that can exponentially yield more by like-minded developers than constraints often found in for-profit orgs; albeit nothing wrong with for-profit, but we are talking just code here.
